src/Saas.SignupAdministration/Saas.SignupAdministration.Web/Services/Email.cs

```diff
@@ -1,52 +1,45 @@
 ﻿using System.Net;
+using System.Net.Http;
 using System.Net.Mail;
+using System.Text.Json;
+using System.Text.Json.Serialization; 
 
 namespace Saas.SignupAdministration.Web.Services
 {
     public class Email : IEmail
     {
+        ILogger<Email> _logger; 
         private readonly EmailOptions _options;
+        private readonly IHttpClientFactory _client; 
 
-        public Email(IOptions<EmailOptions> options)
+        public Email(ILogger<Email> logger, IOptions<EmailOptions> options, IHttpClientFactory client)
         {
+            _logger = logger;
             _options = options.Value;
+            _client = client; 
         }
 
-        public bool Send(string recipientAddress)
+        public async void Send(string recipientAddress)
         {
-           if(!int.TryParse(_options.Port, out int port))
-            {
-                throw new InvalidCastException($"The value of {port} could not be cast to an integer");
-            }
-
-            using SmtpClient client = new(_options.Host, port);
-
-            // TODO: Should update this to use a secure string
-            // These credentails are not secure and will be passed in plain text in this example
-            client.Credentials = new NetworkCredential(_options.Username, _options.Password);
-            MailAddress fromAddress = new(_options.FromAddress);
-            MailAddress toAddress = new(recipientAddress);
-            MailMessage message = new(fromAddress, toAddress);
 
-            message.Subject = _options.Subject;
-            message.Body = _options.Body;
+            var client = _client.CreateClient(_options.EndPoint);
+            JSONEmail email = new JSONEmail();
+            email.HTML = _options.Body;
+            email.Subject = _options.Subject;
+            email.EmailFrom = _options.FromAddress;
+            email.EmailTo = recipientAddress;
+            email.EmailToName = recipientAddress; 
 
+            var json = JsonSerializer.Serialize(email);
+            StringContent content = new StringContent(json, Encoding.UTF8, "application/json");
             try
             {
-                client.Send(message);
+                await client.PostAsync(_options.EndPoint, content);
             }
-            catch
+            catch (Exception ex)
```

Comment:
Add comment here explaining why we're swallowing the error here. ie we don't want the tenant creation to fail because the email failed to send.

Reply:
+1

Reply:
@JasonBergSpur Could you take a look at this?

Reply:
Disregard, I was looking at an outdated version. Sorry 😄